resttest.rb

```diff
@@ -196,33 +196,33 @@ def login_and_check(verb, path)
 
 	def test_login
 		doget("/login", basicauth_joe, 400, "API must be accessed using an HTTP POST method")
-		dopost("/login", nil, basicauth_joe, 200, {:Identity => "joe", :Roles => ["2"]})
+		dopost("/login", nil, basicauth_joe, 200, {:UserId => 1, :Identity => "joe", :Roles => ["2"]})
```

Comment:
For all of these tests, rather than hardcoding the IDs, fetch them from the server. It shouldn't complicate the code much, and it's the right thing to do.